integration/feature/full-run-logs/src/FullRunLogsTests.scala

```diff
@@ -74,5 +74,27 @@ object FullRunLogsTests extends UtestIntegrationTestSuite {
       assert(millProfile.exists(_.obj("label").str == "show"))
       assert(millChromeProfile.exists(_.obj("name").str == "show"))
     }
+    test("failedTasksCounter") - integrationTest { tester =>
```

Comment:
Let's add an example with `--keep-going` to make sure we are correctly handling the `Skipped` case